code/SQLite3Query.php

```diff
@@ -72,7 +72,7 @@ public function numRecords()
         return $c;
     }
 
-    public function nextRecord()
+    public function getIterator()
```

Comment:
https://github.com/silverstripe/silverstripe-framework/blob/5/src/ORM/Connect/MySQLQuery.php#L48Could you add the `#[\ReturnTypeWillChange]` above this method e.g.

Reply:
DONE